src/EditorFeatures/CSharpTest/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProviderTests.cs

```diff
@@ -465,5 +465,31 @@ enum Colors
 ";
             await VerifyItemExistsAsync(markup, "Colors");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task NotAfterDot()
+        {
+            var markup =
+@"namespace ConsoleApplication253
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            M(E.$$) // Completion offers E as a member of E
```

Comment:
Oh.  This comment should be updated. It should be more like: we should *not* offer 'E' off of 'E'.